docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1370,15 +1370,12 @@ duplicates at this time.
 
 
 **Data Errors/Noise** 
-In the future, we will add a noise function designed to replicate 
-missed or incorrect data. This includes incorrect name spelling, 
-addresses, age or DOB, and person swaps. 
 
-Some errors may be introduced within the simulation, such as 
-duplicate people or swapped people. Others may be introduced 
-at the time of sampling, such as name and address misspellings. 
+.. todo::
+    Engineers: these things are not yet defined and might be edited later - they should be parameterizable 
 
-These are not currently modeled. 
+    - We currently have net undercounts, might want to have omissions and duplicates instead. Having a perfect census for now should be fine. 
+    - Creation of a noise function for misrecording data (names, addresses, birthdays) 
```

Comment:
```suggestion
    - Addition of a noise function for misrecording data (names, addresses, birthdays) based only on the true values and the other columns in the survey output
```

Just being overly specific, but we need to specify that our noise function (whatever it is) will not use any currently unobserved values. Or if you don't think we can make this guarantee, I think we need to identify which columns aren't here but we might want to use.

Reply:
Okay, I think that makes sense. I have tried to add in what other parameters are needed (mainly focused on tracked guardians) across the board.